app/src/androidTest/java/org/mozilla/fenix/ui/CollectionTest.kt

```diff
@@ -165,6 +166,7 @@ class CollectionTest {
     }
 
     @Test
+    @Ignore("Failing on https://github.com/mozilla-mobile/fenix/pull/20573")
```

Comment:
nit: Would be better to link to the ticket which in turn can reference where it failed.